apps/api_web/test/api_web/controllers/stop_controller_test.exs

```diff
@@ -206,22 +206,6 @@ defmodule ApiWeb.StopControllerTest do
       end
     end
 
-    test "can include a route we're filtering on", %{conn: conn} do
```

Comment:
This shouldn't be removed: "/stops?filter[route]=Red&include=route` should work.

Reply:
Ok, it's just that swagger didn't had "route" as include options, so I though it was obsolete. Fixed.

Reply:
Can you add that, either as a part of this or in a new ticket?

Reply:
I've added it to the list of includes in previous commit -- https://github.com/mbta/api/pull/11/commits/0bce0c32c06bef24ff50f09d7e68fe2610ef366f#diff-cee2a701bc0c45852758feec0c9de777L9 -- and it seems to be working without requiring additional code changes: 

```bash
levs-mbp:~ lev$ curl 'http://localhost:4000/stops?route=Mattapan&include=route' | jsonpp
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4426  100  4426    0     0  85970      0 --:--:-- --:--:-- --:--:-- 86784
{
  "data": [
    {
      "attributes": {
        "address": "Dorchester Ave and Ashmont St, Boston, MA 02124",
        "description": null,
        "latitude": 42.284652,
        "location_type": 1,
        "longitude": -71.064489,
        "name": "Ashmont",
        "platform_code": null,
        "platform_name": null,
        "wheelchair_boarding": 1
      },
      "id": "place-asmnl",
      "links": {
        "self": "/stops/place-asmnl"
      },
      "relationships": {
        "child_stops": {},
        "facilities": {
          "links": {
            "related": "/facilities/?filter[stop]=place-asmnl"
          }
        },
        "parent_station": {
          "data": null
        },
        "route": {
          "data": {
            "id": "Mattapan",
            "type": "route"
          }
        }
      },
      "type": "stop"
    }
[...]
```

Reply:
Sorry, I meant to add it to Swagger.

Reply:
Adding to Swagger is what I did in the commit linked above. 

![image](https://user-images.githubusercontent.com/45011335/53508930-c2392900-3a88-11e9-9feb-096249b72c69.png)